src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/ResourceActions/ResourceActions.ts

```diff
@@ -61,7 +66,9 @@ export var resourceActionsDirective = (
         scope: {
             resourcePath: "@",
             parentPath: "=?",
+            processUrl: "@?",
```

Comment:
`processUrl` is really not a good  name for this as it does not need to be a process. Proposal: `badgeRoot`.